### ISSUE TITLE
Tlog-play fast forward: What happen if the time fast-forward  < current time. 

### ISSUE THREAD

**Issue by danghai** [Scribery/tlog#159]
What happen if the time fast-forward < current time. For example: at the current time: '40', and then I want to fast-forward '30'. I see it stops and then re-run at '40'. It should be not effect if fast-forward time < current time.

**Comment by spbnick** [comment]
Yes, it appears a legitimate issue, thank you Hai. However, could you please keep the issue subject to the point, descriptive, and brief. Try to make it easier to read a list of issues. Thank you!

**Comment by danghai** [comment]
Okay, thank you fix my issue subject.

**Comment by danghai** [comment]
I have one idea to handle this issue, but it does not work now. My idea: 
After 'tlog_timestr_parser_yield', we will get new value 'tlog_play_goto_ts' from user. And then, I check this value with 'tlog_play_local_last_ts' ( Local time of packet output last), if it is less than, we need to turn off the flag 'tlog_play_goto_active'. Unfortunately, it does not work now. Or I may misunderstand the variable 'tlog_play_local_last_ts'. You can see it under commit: 
https://github.com/danghai/tlog/commit/17d9ddcb0eeead06ad65f91e60170f19e67edb25
